src/libmongoc/src/mongoc/mongoc-timeout-private.h

```diff
@@ -22,13 +22,15 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+#include <bson/bson.h>
+
```

Comment:
Does this PR need any of these changes to the timeout struct? Since we aren't using it anymore I think the changes can be reverted.

Reply:
I will remove the changes to the timeout struct.